lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.html

```diff
@@ -236,11 +236,14 @@ <h2>Do better web tester page</h2>
 <button class="small-button">Do something</button>
 <button class="small-button">Do something else</button>
 
-<!-- Some websites overwrite the original Error object. The captureJSCallUsage function
-  relies on the native Error object and prepareStackTrace from V8. When overwriting the stack
-  property the e.stack inside the gatherer won't be in the correct format
-  https://github.com/GoogleChrome/lighthouse/issues/1194 -->
-<script>window.Error = function(error) { this.stack = 'stacktrace'; };</script>
+<!-- Some websites overwrite the original Error object. That causes problems if Lighthouse uses
+  the `prepareStackTrace` API. https://github.com/GoogleChrome/lighthouse/issues/1194
+  However, since https://github.com/GoogleChrome/lighthouse/pull/5509,
+  we haven't used that API. So, no need to replicate the problem anymore. This comment only remains
+  as documentation for the legacy __nativeError.
```

Comment:
do we even need that? feels like a comment on this PR that removes it could suffice :)